scripts/ci/qemu-run.sh

```diff
@@ -52,8 +64,12 @@ case "$2" in
 	vnc)
 		qemu-system-x86_64 $q35_params -nographic -vnc :0
 		;;
+	install_ubuntu)
+		qemu-system-x86_64 $q35_params -cdrom qemu-data/ubuntu.iso
+		;;
 	*)
-    qemu-system-x86_64 $q35_params
+    qemu-system-x86_64 $q35_params -display gtk,window-close=off -vga virtio
```

Comment:
@macpijan this conflicts with vnc feature, I'm not running QEMU locally but on remote machine and would like to vnc to it.

Reply:
But vnc is another command, so it would not be used. This gives proper resolution and fullscreen when running graphical interface locally. Not sure if all parameters are needed TBH. We may still need to work on this, I took some known working configuration and based on that to be able to install OS.

Reply:
@pietrushnic Can you elaborate here?

Reply:
@pietrushnic But you used the `vnc` branch of this switch, not `*`, or not?

Reply:
I reworked the script a bit more